scripts/SANS/sans/state/data.py

```diff
@@ -134,17 +134,17 @@ def build(self):
         # There are some elements which need to be read from the file. This is currently:
```

Comment:
Does this comment imply that reading will still be done here? Am I to understand correctly that the file reading has actually now already been done?

Reply:
Yes the reading has now already been done outside the state builder function. I've updated the comment to make this clearer.